ntp-daemon/src/tracing.rs

```diff
@@ -60,6 +70,11 @@ pub fn init(
             }
         }
 
-        Ok(guard)
+        let state = TracingState {
+            guard,
+            reload_handle: fmt_handle,
+        };
+
+        Ok(state)
```

Comment:
NIT: prefer Ok(TracingState {}) instead of a named intermediate, as it makes the code more compact and readable.